.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node_version: [14.x, 16.x]
+        node_version: [16.x]
```

Comment:
This will be done in a separate PR as it will require to upgrade to vite.js and maybe other npm modules upgrade.
It is not different from the current state that it is being built in node 16, vsix is packed and can be run in BAS.